README.rst

```diff
@@ -37,6 +38,43 @@ for a complete list of features, flags, and documentation.
 jobs
 ^^^^
 
+create
+~~~~~~
+
+Create an data export job request and return the export job id.  There are two
+general workflows for creating data export requests.  To create a data export
+requests for all available schemas for a course::
+
+    courseraresearchexports jobs create schemas --courseId $COURSE_ID --statementOfPurpose "testing data export"
```

Comment:
can we make our examples use `courseSlug`? it's more accessible for people in general.